Makefile

```diff
@@ -54,7 +54,7 @@ generate-code:
 	$(info Generating deepcopy...)
 	${BIN_DIR}/deepcopy-gen -i ./pkg/apis/controller/v1beta1/ -O zz_generated.deepcopy 
 
-images: verify-tag-name
+images: verify-tag-name mcad-controller
```

Comment:
The build instructions map is `build.sh` => `make mcad-controller` and images.sh => `make images`.  If this `mcad-controller` target is added here probably the instructions should be updated to remove running the `build.sh`?  To be consistent, likely `images-podmad` target should be updated?

Reply:
Got it, will update the documentation.

Reply:
Great, thanks!